File: Python/film.py
class Film:
    def __init__(self, id: int, judul: str, genre: str, jam_tayang: str, gambar: str):
        self.__id = id
        self.__judul = str(judul)
        self.__genre = str(genre)
        self.__jam_tayang = str(jam_tayang)
        self.__gambar = str(gambar)

    # Getter
    def getId(self) -> int:
        return self.__id

    def getJudul(self) -> str:
        return self.__judul

    def getGenre(self) -> str:
        return self.__genre

    def getJamTayang(self) -> str:
        return self.__jam_tayang

    def getGambar(self) -> str:
        return self.__gambar

    # Setter
    def setJudul(self, judul: str) -> None:
        self.__judul = str(judul)

    def setGenre(self, genre: str) -> None:
        self.__genre = str(genre)

    def setJamTayang(self, jam_tayang: str) -> None:
        self.__jam_tayang = str(jam_tayang)

    def setGambar(self, gambar: str) -> None:
        self.__gambar = str(gambar)

    def tampilkan(self) -> None:
        print(f"ID           : {self.__id}")
        print(f"Judul        : {self.__judul}")
        print(f"Genre        : {self.__genre}")
        print(f"Jam Tayang   : {self.__jam_tayang}")
        print(f"Gambar       : {self.__gambar}")
        print("-----------------------------")

File: Java/Film.java
public class Film {
    private int id;
    private String judul;
    private String genre;
    private String jamTayang;
    private String gambar;

    // Constructor kosong
    public Film() {}

    // Constructor berparameter
    public Film(int id, String judul, String genre, String jamTayang, String gambar) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
        this.jamTayang = jamTayang;
        this.gambar = gambar;
    }

    // Getter
    public int getId() { return id; }
    public String getJudul() { return judul; }
    public String getGenre() { return genre; }
    public String getJamTayang() { return jamTayang; }
    public String getGambar() { return gambar; }

    // Setter
    public void setJudul(String judul) { this.judul = judul; }
    public void setGenre(String genre) { this.genre = genre; }
    public void setJamTayang(String jamTayang) { this.jamTayang = jamTayang; }
    public void setGambar(String gambar) { this.gambar = gambar; }

    public void tampilkan() {
        System.out.println("ID           : " + id);
        System.out.println("Judul        : " + judul);
        System.out.println("Genre        : " + genre);
        System.out.println("Jam Tayang   : " + jamTayang);
        System.out.println("Gambar       : " + gambar);
        System.out.println("-----------------------------");
    }
}
